tensorflow/lite/micro/kernels/arc_mli/conv.cc

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+/* Copyright 2020-2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
our license check only accepts a single year. Please update to be `Copyright 2021 ...`